.github/workflows/gh-pages.yml

```diff
@@ -36,12 +36,13 @@ jobs:
         id: cache-docs
         with:
           path: docs/_examples_database/
-          key: docs_database_and_stim_results_cache
+          # See https://github.com/actions/cache/blob/main/tips-and-workarounds.md#update-a-cache
+          key: docs_database_and_stim_results_cache-${{ github.run_id }}
```

Comment:
@nelimee we don't want to create a cache for every run, right? 

![image](https://github.com/user-attachments/assets/a38dce69-5949-4358-9aeb-7fe51c21f792)

Reply:
> a new cache is created at each workflow run,

This is going to increase the docs build time for the first time.

Reply:
> > a new cache is created at each workflow run,
> 
> This is going to increase the docs build time for the first time for each run.

No, explanations in my comment below.